saga/inMemorySagaLog.go

```diff
@@ -69,3 +69,19 @@ func (log *inMemorySagaLog) GetMessages(sagaId string) ([]sagaMessage, error) {
 		return nil, nil
 	}
 }
+
+/*
+ * Returns all Sagas Started since this InMemory Saga was created
+ */
+func (log *inMemorySagaLog) GetActiveSagas() ([]string, error) {
+	log.mutex.RLock()
```

Comment:
defer has a pretty high performance cost, given how simple this code is i just chose to explicitly place it